.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,12 @@
+name: Integration tests
+
+on:
+  pull_request:
+
+jobs:
+  integration-tests:
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    secrets: inherit
+    with:
+      juju-channel: 3.1/stable
+      provider: lxd
```

Comment:
what about using the edge self-hosted runners? ;-)

Reply:
Let me add them on the follow up [PR](https://github.com/canonical/tmate-ssh-server-operator/pull/5)